syntax = "proto3";

option go_package = "/client;main";

service MessageService { rpc Ping(PingRequest) returns (PingResponse); }

message PingRequest { string message = 1; }

message PingResponse { string reply = 1; }
